src/Modules/OrchardCore.Commerce/Controllers/WebhookController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using OrchardCore.Commerce.Abstractions;
+using OrchardCore.Commerce.Models;
+using OrchardCore.Entities;
+using OrchardCore.Settings;
+using Stripe;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Commerce.Controllers;
+
+[Route("stripe-webhook")]
+[ApiController]
+[Authorize(AuthenticationSchemes = "Api"), IgnoreAntiforgeryToken, AllowAnonymous]
+public class WebhookController : Controller
+{
+    // This is a default test Stripe CLI webhook secret for testing your endpoint locally. It can be shown, as this is
+    // the same always for testing in Stripe.
+    private const string LocalEndPointSecret =
+        "whsec_453d1046fc31377b7a93e82b839c9e6e065d7117b6e02422e55eac99da087463";
```

Comment:
Where does this secret come from? I can't find it anywhere in the documentations and when I run `stripe` it shows a different secret. Please include a link

Reply:
Okay, so it might have been a generated local test key for me as well. It's coming from here:
[https://dashboard.stripe.com/test/webhooks/create?endpoint_location=local](https://dashboard.stripe.com/test/webhooks/create?endpoint_location=local)